portfolio/src/main/webapp/index.html

```diff
@@ -14,14 +14,16 @@ <h1>a page about me</h1>
 
     <div id="body-content">
         <div id="about" class="sidebar">
-            <div id="links" class="box">
+            <div id="buttons" class="box">
```

Comment:
Does the id of this div need to change? Seems like it's still links and not buttons?

Reply:
i added another 'button' to generate some random quotes using javascript, i just wanted the id to be more descriptive because they werent just links anymore in that div (line 26 is where i added the quote generator)